voxmeld/src/cmd/voxmeld/main.go

```diff
@@ -3,15 +3,44 @@ package main
 import (
 	"bytes"
 	"encoding/binary"
+	"io/fs"
+	"path/filepath"
 	"strings"
+	"sync"
+	"sync/atomic"
 
 	//	"fmt"
+	"fmt"
 	"log"
 	"os"
-	"path/filepath"
+	"runtime"
 	"strconv"
+	"time"
 )
 
+var VERSION = "v0.4.3"

```

Comment:
For a future release: We could also consider getting the version from the environment - that way we won't have to rebuild all EXEs for a new release. Something like:
```go
version := os.GetEnv("VERSION")
if version == "" {
  log.Fatal("VERSION missing")
}
```